packages/framework/src/DataFixtures/Base/CategoryRootDataFixture.php

```diff
@@ -20,36 +19,35 @@ class CategoryRootDataFixture extends AbstractReferenceFixture implements Depend
     protected $categoryFactory;
 
     /**
-     * @var \Shopsys\FrameworkBundle\Model\Category\CategoryDomainFactoryInterface
+     * @var \Shopsys\FrameworkBundle\Model\Category\CategoryDataFactory
      */
-    protected $categoryDomainFactory;
+    protected $categoryDataFactory;
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Category\CategoryFactoryInterface $categoryFactory
-     * @param \Shopsys\FrameworkBundle\Model\Category\CategoryDomainFactoryInterface $categoryDomainFactory
+     * @param \Shopsys\FrameworkBundle\Model\Category\CategoryDataFactory $categoryDataFactory
      */
     public function __construct(
         CategoryFactoryInterface $categoryFactory,
-        CategoryDomainFactoryInterface $categoryDomainFactory
+        CategoryDataFactory $categoryDataFactory
     ) {
         $this->categoryFactory = $categoryFactory;
-        $this->categoryDomainFactory = $categoryDomainFactory;
+        $this->categoryDataFactory = $categoryDataFactory;
     }
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
      */
     public function load(ObjectManager $manager)
     {
-        $rootCategory = $this->categoryFactory->create(new CategoryData());
+        $categoryData = $this->categoryDataFactory->createDefault();
+        $categoryData->enabled = [
+            Domain::FIRST_DOMAIN_ID => true,
```

Comment:
Shouldn't this behavior be a part of CategoryDataFactory::createDefault()?

Reply:
well not really, `createDefault()` creates categoryData with enabled set to false. So this line makes sure that category will be enabled on domain

Reply:
Why is it created with `false`s as default? Isn't more appropriate to create it with `true`s? On `master`, all domains are enabled by default when eg. creating a new category:

![image](https://user-images.githubusercontent.com/10008612/41303387-685a3e20-6e6d-11e8-921b-0124a121f5d3.png)

Reply:
Okay, i think that i handled all of our enabled domain properties and it is set to be enabled as default, which made data fixtures much more easier.